layouts/shortcodes/get-service-checks-from-git.html

```diff
@@ -33,10 +33,11 @@
 
     <p>
       <strong>{{ .check }}</strong><br>
-      {{ .description | markdownify }}
+      {{ .description | markdownify }}<br>
+      <i>Statuses: {{ range $i, $e := .statuses }}{{ if $i }}, {{ end }}{{ $e }}{{ end }}</i>
```

Comment:
I didn't get any errors running this but i was going to suggest doing a with on it so that it .statuses doesn't exist for some reason it won't break.

```suggestion
      {{ with .statuses }}<i>Statuses: {{ range $i, $e := . }}{{ if $i }}, {{ end }}{{ $e }}{{ end }}</i>{{ end }}
```